packages/FormElement/src/components/Label/Label.js

```diff
@@ -9,32 +9,35 @@ const propTypes = {
   hasOptionalLabel: PropTypes.bool.isRequired,
   hasRequiredLabel: PropTypes.bool.isRequired,
   help: PropTypes.node,
-  id: PropTypes.string.isRequired,
+  id: PropTypes.string,
   isInline: PropTypes.bool.isRequired,
   isVisuallyHidden: PropTypes.bool.isRequired,
-  label: PropTypes.string.isRequired,
+  labelText: PropTypes.node.isRequired,
```

Comment:
Allowing label text content to be rendered as html now

Reply:
Why not keep the original prop name? Maintaining the same name means it's not a breaking change, and postfixing with `Text` is probably less intuitive, since it's now _anything_, not just text, it could be any html or jsx.

Reply:
It's not a breaking change because the Label component is rendered by the formElement, not the consuming application. But sure, I can see why text would make you think of "string" only. I'll change it back though although something about just "label" seems to value as this is a "Label" component (which may also render a `legend` element incidentally)